src/js/components/service-requests/service-request-detail.js

```diff
@@ -19,92 +24,176 @@ export class ServiceRequestDetail {
 
-    this.$element = new FullWidthGrid([$loadingPlaceholder]).render();
-
-    api
-      .getServiceRequest(serviceRequestId)
-      .then((response) => {
-        $loadingPlaceholder.remove();
-
-        const $serviceRequestInfoHeading = $sectionHeading.clone();
-        const $serviceDescriptionHeading = $sectionHeading.clone();
-
-        $serviceRequestInfoHeading
-          .find(".section-title")
-          .text("Service request information");
-
-        $serviceDescriptionHeading
-          .find(".section-title")
-          .text("Service request description");
-
-        this.$element.append($serviceRequestInfoHeading);
+    const {
+      $uploadImagesInput,
+      $uploadImagesClass,
+      $uploadImagePreviewTemplate,
+    } = createImageFormFields();
+
+    let uploadedFiles = {};
+
+    function handleInputFilesChanged() {
+      updateUploadedFiles(
+        this.files,
+        uploadedFiles,
+        $uploadImagePreviewTemplate,
+        $uploadImagesClass
+      );
+    }
+    $uploadImagesInput.change(handleInputFilesChanged);
+
+    const $submitButton = $(".components .button.button--form-submit")
+      .clone()
+      .attr({
+        value: "Upload Images",
+        id: "submit-images",
+      });
 
-        this.$element.append(
-          new BasicBlock({
-            title: "Date submitted",
-            subtitle: response.request_date
-              ? getDateString(response.request_date, false)
-              : "No date provided",
-          }).render()
-        );
+    $submitButton.hide();
+
+    $submitButton.on("click", (event) => {
+      event.preventDefault();
+      if (Object.keys(uploadedFiles).length === 0) {
+        alert("Please select one or more images to upload.");
+        return;
+      }
+
+      $submitButton.attr({ value: "Uploading...", disabled: true});
+
+      api
+        .submitServiceRequestFiles(
+          serviceRequestId,
+          getFormDataFromArray(uploadedFiles)
+        )
+        .then(() => {
+          window.location.reload();
+        })
+        .fail((a, b) => {
+          this.$element.empty().append(
+            new StatusMessage({
+              text: "Error while adding files to the service request.",
+              status: "failure",
+            }).render()
+          );
+          console.error(a, b);
+          $submitButton.attr({ value: "Upload Images", disabled: false});
+        });
+    });
 
-        this.$element.append(
-          new BasicBlock({
-            title: "Reference number",
-            subtitle: response.on_premis_reference || "No reference number yet",
-          }).render()
-        );
+    this.$element = new FullWidthGrid([$loadingPlaceholder]).render();
 
+    api.getServiceRequest(serviceRequestId).then((response) => {
+      $loadingPlaceholder.remove();
+
+      const $serviceRequestDetail = $(
+        ".components .rich-text.rich-text--basic-block.w-richtext"
+      ).clone();
```

Comment:
This element is not actually being used anymore.

Reply:
I noticed it was not being used prior to this PR but left it as it wasn't within the scope of this PR.

I can remove it though since we are looking at it now.

Reply:
I've removed it